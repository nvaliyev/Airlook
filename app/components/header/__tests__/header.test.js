import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Header from '..';
describe(' Header', () => {
    test('Header Snap', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(< Header/>);
        expect(snapshot).toMatchSnapshot();
    });
});


