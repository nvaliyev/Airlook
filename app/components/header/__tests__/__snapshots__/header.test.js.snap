// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Header Header Snap 1`] = `
<LinearGradient
  colors={
    Array [
      "#000000d9",
      "#5d6fc4",
      "#6f347c",
      "#000000d9",
    ]
  }
  end={
    Object {
      "x": 1,
      "y": 0,
    }
  }
  start={
    Object {
      "x": 0,
      "y": 3,
    }
  }
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "height": 60,
      "justifyContent": "space-between",
    }
  }
>
  <Burger />
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "paddingHorizontal": 5,
      }
    }
  >
    <View
      style={
        Object {
          "flexDirection": "row",
          "paddingHorizontal": 1,
        }
      }
    >
      <CurrTime />
    </View>
    <View
      style={Object {}}
    >
      <Themed.Icon
        color="#F7F9FC"
        name="search"
        size={30}
      />
    </View>
  </View>
</LinearGradient>
`;
